app/components/account.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class AccountComponent extends Component {
+    @tracked actNum = '980 528 896';
```

Comment:
check with the team if we have any specific format for accountnumber and update

Reply:
changed the accountNumber format

Reply:
still some requirement is not satisfied..check with backend team for format

Reply:
Have you checked with backend team, I have informed them that account number should start with 90 and have 12 digits